project/Dependencies.scala

```diff
@@ -32,7 +32,7 @@ object Dependencies {
   val dataFormat = "com.fasterxml.jackson.dataformat" % "jackson-dataformat-cbor" % jacksonVersion
   val bcprovJdk15on = "org.bouncycastle" % "bcprov-jdk15on" % "1.60"  //-- added explicitly - snyk report avoid logback vulnerability
   // This is required to force aws libraries to use the latest version of jackson
-  val jacksonDataBind =  "com.fasterxml.jackson.core" % "jackson-databind" % jacksonVersion
+  val jacksonDataBind =  "com.fasterxml.jackson.core" % "jackson-databind" % (jacksonVersion + ".1")
```

Comment:
why the `+ ".1"` here?

Reply:
there is a "patch" version that we need, but they didn't release the full set of jackson libraries for the patch
![image](https://user-images.githubusercontent.com/7304387/114202828-7f2c2e00-994f-11eb-9ee3-86064e4d0d9f.png)

Reply:
the next time it's bumped, the .1 can be removed again (it won't compile otherwise)